x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -299,15 +300,15 @@ export type BulkEditOptions<Params extends RuleTypeParams> =
   | BulkEditOptionsFilter<Params>
   | BulkEditOptionsIds<Params>;
 
-export interface BulkDeleteOptionsFilter {
+export interface BulkCommonOptionsFilter {
   filter?: string | KueryNode;
 }
 
-export interface BulkDeleteOptionsIds {
+export interface BulkCommonOptionsIds {
   ids?: string[];
 }
 
-export type BulkDeleteOptions = BulkDeleteOptionsFilter | BulkDeleteOptionsIds;
+export type BulkCommonOptions = BulkCommonOptionsFilter | BulkCommonOptionsIds;
```

Comment:
Trying to understand the reasoning of this renaming. What does `common` mean in this context?